examples/README.md

```diff
@@ -6,46 +6,59 @@ This directory contains many examples of different ways to use Bokeh. As Bokeh h
 fast, it is important that you **ensure that the version of an example you're looking at matches
 the version of Bokeh you are running**.
 
-### [`app`](app/)
+### [`advanced`](advanced/)
 
-This directory contains examples of Bokeh Apps, which are simple and easy to create web applications for data visualization or exploration.
+This folder contains examples of more advanced or complex Bokeh applications and use cases, beyond the basic plotting and visualization examples that are included in the examples folder.
 
-### [`custom`](custom/)
+### [`basic`](basic/)
 
-The content of this directory demonstrate how to add custom extensions to Bokeh. Check project's documentation [page](https://docs.bokeh.org/en/latest/docs/user_guide/extensions.html)
+Each example in the basic folder provides a clear and concise demonstration of how to create a specific type of visualization using Bokeh. These examples are intended for new users who are just getting started with Bokeh and want to learn how to create basic visualizations using the library.
 
-### [`embed`](embed/)
+### [`integration`](integration/)
```

Comment:
Actually the "integration" directory contains examples that are used as integration tests, they will probably be moved at some point. I think this entire section can just be omitted.